packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -138,6 +133,11 @@ export function SchemaDrivenAnnotationEditor(
     useErrorBoundary: true,
   })
 
+  // Annotation fields fetched and modified via the form
+  const [formData, setFormData] = React.useState<
+    Record<string, unknown> | undefined
+  >(annotations)
```

Comment:
Note the initial value is now `annotations`, to prevent a case where this was `undefined` which broke the form component